scripts/roachprod-get-latest.sh

```diff
@@ -1,31 +1,37 @@
 #!/usr/bin/env bash
 
-#Downloads the latest roachprod binary from teamcity to the specified directory
+# Downloads the _latest_ roachprod binary (on master) from GCS to the specified directory
 # or the current directory if none is specified.
-# This does NOT use the TeamCity REST API, but rather a direct download link for the latest successful build
-# for the current platform.
+# Assumes that you have gsutil installed and authenticated; see [1] for those details.
+#
+# The specified directory, release branch, OS and CPU architecture are optional arguments. They can be overridden by
+# simultaneously specifying _all_ in the described order.
+#
+# [1] https://cockroachlabs.atlassian.net/wiki/spaces/TE/pages/144408811/Roachprod+Tutorial
 
-TC_BASE="https://teamcity.cockroachdb.com/repository/download"
-TC_ARTIFACT_PATH="latest.lastSuccessful/bazel-bin/pkg/cmd/roachprod/roachprod_/roachprod"
+DEFAULT_BRANCH="master"
+DEFAULT_OS=$(uname | tr '[:upper:]' '[:lower:]')
+DEFAULT_ARCH=$(arch | tr '[:upper:]' '[:lower:]')
+BUCKET="cockroach-nightly"
```

Comment:
Is there any garbage collection set up, or is the idea that we would be holding on to all of these roachprods indefinitely? Each is ~100MB. Storage costs are going to start adding up quickly.

Reply:
We could configure a retainment policy on the bucket... but considering this is one binary per release, per night, then our annual cost is ~$28, using crude arithmetic `(4* 100 * 365) / 1024) * 0.2`. So, it's not that expensive :)